docs.kosli.com/netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
   publish = "public"
-  command = "HUGO_BASEURL=$URL hugo"
+  command = "git fetch --tags -f && bash update_version.sh && HUGO_BASEURL=$STAGING_URL hugo"
```

Comment:
this uses git tags to check for version which works most of the time, but has some short comings:
- if release CI fails, you may have a version (from the tag) that does not exist as a release yet
- this also captures draft-releases, release candidates etc as versions, while we probably want only stable releases.

This line gives you a guaranteed stable version:
```
gh release list --repo kosli-dev/cli -L 1 --exclude-pre-releases --exclude-drafts | awk '{print $1}'
```